content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/150-using-prisma-migrate.mdx

```diff
@@ -39,6 +39,91 @@ model User {
 }
 ```
 
+<SwitchTech technologies={['*', 'postgresql']}>
+
+To map your data model to the database schema, you need to use the `prisma migrate` CLI commands:
+
+```terminal
+npx prisma migrate dev --name init
+```
+
+This command does two things:
+
+1. It creates a new SQL migration file for this migration
+1. It runs the SQL migration file against the database
+
+> **Note**: `generate` is called under the hood by default, after running `prisma migrate dev`. If the `prisma-client-js` generator is defined in your schema, this will check if `@prisma/client` is installed and install it if it's missing.
+
+</SwitchTech>
+
+<SwitchTech technologies={['*', 'mysql']}>
+
+To map your data model to the database schema, you need to use the `prisma migrate` CLI commands:
+
+```terminal
+npx prisma migrate dev --name init
+```
+
+This command does two things:
+
+1. It creates a new SQL migration file for this migration
+1. It runs the SQL migration file against the database
+
+> **Note**: `generate` is called under the hood by default, after running `prisma migrate dev`. If the `prisma-client-js` generator is defined in your schema, this will check if `@prisma/client` is installed and install it if it's missing.
+
+</SwitchTech>
+
+<SwitchTech technologies={['*', 'planetscale']}>
+
+To use Prisma Migrate with PlanetScale, you will also need to add the [`referentialIntegrity` preview feature](/concepts/components/prisma-schema/relations/referential-integrity) to your `schema.prisma` file:
+
+```prisma file=schema.prisma highlight=3;add|9;add
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["referentialIntegrity"]
+}
+
+datasource db {
+  provider             = "mysql"
+  url                  = env("DATABASE_URL")
+  referentialIntegrity = "prisma"
+}
+```
```

Comment:
This whole section _could_ already be part of the "connect to your database" section as it is not only required when using Prisma Migrate, but also to tell Prisma Client to handle referential integrity (via referential actions) when e.g. deleting data.

Reply:
Moved.

Reply:
Don't think this made it - Percy still shows it on the "Using Prisma Migrate" page.